docs/source/index.rst

```diff
@@ -84,14 +84,18 @@ environment but rather for experimentation, learning, or new feature development
 Its most common use would be to interact with the streamz example jupyter notebooks. Lets walk through the steps needed for this.
 
 - Build the Docker container
-.. code-block:: bash
-$ docker/build.sh
+
+  .. code-block:: bash
```

Comment:
I don't think this directive (and below) should be indented

Reply:
It does, that was the problem :) That is how you indicate that the code block is part of the list item.